src/main/content/robots.txt

```diff
@@ -1,3 +1,3 @@
 User-agent: *
-Disallow: /blog/redirected.html
-Noindex: /blog/redirected.html
```

Comment:
I don't know why it's saying that I changed this...here's a screenshot of this file in master currently
![Screen Shot 2019-10-16 at 11 25 52 AM](https://user-images.githubusercontent.com/37549026/66934028-de81da00-f007-11e9-905f-3e004100c739.png)

Reply:
its talking about https://github.com/kabanero-io/kabanero-website/blob/master/src/main/content/robots.txt and your screen shot is the robots.txt on the root (there's 2 of them)